principal_travel/settings.py

```diff
@@ -39,8 +39,11 @@
     'django.contrib.staticfiles',
     'rest_framework',
     'travel',
+    'registration',
+    'designstandards'
 ]
 
+REGISTRATION_AUTO_LOGIN = True # Automatically log the user in.
```

Comment:
Can you add a comment above this to indicate that this is associated with #django-registration-redux? It would be helpful information-wise, but also to mark out a section of the settings where any future/additional settings for this module would be kept.